app/assets/javascripts/games/rps.js

```diff
@@ -1,3 +1,72 @@
-$(function() {
```

Comment:
@rrykowski - I would suggest adding this back, it is a way in jQuery to ensure the javascript is not executed until the page is actually ready. If you don't wrap javascript that relies on the page having been fully rendered by the browser before it executes javascript code that may depend on the full rendering of the page, you can run into unexpected behavior. Let's make sure we come back to the topic of the DOM and what some of the events are that occur once an html file is returned to the browser by the server.